src/args.rs

```diff
@@ -144,6 +147,22 @@ pub struct Args {
     pub max_file_size_unencrypted_mb: usize,
 }
 
+impl From<Args> for DatabaseArgs {
```

Comment:
The curse of backwards compatibility. Good solution!

Reply:
Thanks. Yeah thought why not use some language features to make it nice :)